src/structures/BaseManager.ts

```diff
@@ -15,4 +28,34 @@ export abstract class BaseManager {
             includes: this.INCLUDES,
         };
     }
+
+    /**
+     * Fetches rach page and joins the results.
+     * @returns Dictionary of the specified types
+     * @internal
+     */
+    protected async getFetchAll<T, K>(...options: unknown[]): Promise<Dict<T, K>> {
+        type Data = Dict<T, K>;
```

Comment:
There's no need for this type declaration, you can specify `<T, K>` in the `data` variable and TS should infer from there.

Reply:
What do you mean? `getFetchAll<T, K>` is necessary, to add the 2 types to context. I can't remove that.

Reply:
I mean `type Data = Dict<T, K>;`, you can cast the `data` and `page` variables to `Dict<T, K>` instead of using a new type declaration to do it.